index.html

```diff
@@ -14,7 +14,7 @@
 
   <div class="index-container">
     <section class="diary">
-      <table id="breakfast-list">
+      <table class="breakfast-list" id="400">
```

Comment:
@drod1000, keeping the totals here for now gets the job done. We can put them into the totals table later.